docstore/query.go

```diff
@@ -38,7 +38,7 @@ func (c *Collection) Query() *Query {
 
 // Where expresses a condition on the query.
 // Valid ops are: "=", ">", "<", ">=", "<=, "in", "not-in".
-// Valid values are strings, integers, floating-point numbers, and time.Time values.
+// Valid values are strings, integers, floating-point numbers, time.Time and boolean (only for "=", "in" and "not-in") values.
```

Comment:
It currently doesn't work for "in" or "not-in", please fix.

Reply:
I have added a test, and it's working for Firestore, MongoDB, but I don't have access to AWS, so I wasn't able to test it

Reply:
Ah, it's working. My attempt used `true` not `[]bool{true}` for the `not-in`. Thanks!